guides/common/modules/proc_loading-the-default-openscap-contents.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Loading_the_Default_OpenSCAP_Contents_{context}"]
+= Loading the Default OpenSCAP Contents
+
+Use this procedure to load the default OpenSCAP contents nn {ProjectServer}.
+This procedure ensures that data streams available in `/usr/share/xml/scap/ssg/content/` are loaded and assigned to all organizations and locations.
+
+.Prerequisites
+* Your user account has the `create_scap_contents` permission.
+* Ensure that the *SCAP Security Guide* package is installed on {ProjectServer}:
++
+ifdef::foreman-deb[]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {package-install} ssg-base ssg-debderived ssg-debian ssg-nondebian ssg-applications
+----
+endif::[]
+ifndef::foreman-deb[]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {package-install} scap-security-guide
```

Comment:
I'm wondering if this should be solved either in our installer or at a packaging level. You may not know this, but the general question to ask: how many people follow this step?
* If it's all users of `foreman_openscap` then it should be a packaging dependency.
* If it's most and it's a small package, then I'd still make it a packaging dependency. 
* Otherwise I'd lean to an installer option.

Not a blocker to updating the docs, but IMHO an issue to file as a follow up. Docs are always a good moment to reflect on the entirety of the steps taken.